src/ripple/app/paths/impl/DirectStep.cpp

```diff
@@ -211,7 +212,8 @@ DirectStepI::redeems (ReadView const& sb, bool fwd) const
 {
     if (!fwd)
     {
-        auto const srcOwed = creditBalance2 (sb, dst_, src_, currency_);
+        auto const srcOwed = accountHolds (
```

Comment:
Removing these two calls to `creditBalance2` leaves it unused; unless you plan on reusing it again in the future, it should be removed.